components/atom/button/src/index.scss

```diff
@@ -0,0 +1,139 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$c-button: $c-primary !default;
+$c-button-light: color-variation($c-primary, 4) !default;
+$c-button-dark: $c-primary-dark !default;
+$c-button-light-contrast: color-variation($c-primary, 3) !default;
+$c-button-white: #ffffff !default;
```

Comment:
I'm not sure if we should call it `...-white`, maybe in a vertical's theme they use another color as the foreground color.  Also, you will move this vars to sui-theme, right?

Reply:
Probably will be migrated. But It's one of the discussions I wanna have with the team